src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.targets

```diff
@@ -0,0 +1,31 @@
+<Project>
+  <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' ">
+    <ProjectCapability Include="UseMauiCore" />
+    <ProjectCapability Condition=" '$(UseMaui)' == 'true' " Include="UseMaui" />
+  </ItemGroup>
+  <Import
+      Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' or '$(UseMauiEssentials)' == 'true' or '$(UseMauiAssets)' == 'true') "
+      Project="Sdk.targets" Sdk="Microsoft.Maui.Sdk-$(TargetFrameworkVersion)"
+  />
```

Comment:
Load the correct aliased version of the SDK and it will resolve to the correct pack version.